exasol/toolbox/tools/security.py

```diff
@@ -313,7 +312,7 @@ def json_issue_to_markdown(
         json_file: typer.FileText = typer.Argument(default="", mode="r", help="json file with issues to convert"),
 ) -> None:
     content = json_file.read()
-    issues = from_json(content)
+    issues = from_json(content, Path(__file__).parent.parent.parent.parent)
```

Comment:
This path/prefix should not be hardcoded, lets discuss it real quick. The rest of he PR looks good!